modules/web/src/assets/css/global/_main.scss

```diff
@@ -714,3 +714,101 @@ km-cni-version {
     margin-top: 30px;
   }
 }
+
+// side-nav-field content styling
+
+.collapse-sidenav {
+  overflow-x: hidden;
+  width: variables.$sidenav-width-collapsed !important;
+
+  .menu-item a i {
+    margin-right: 0;
+  }
+}
+
+.sub-menu-item {
+  position: relative;
+  z-index: 1;
+
+  a {
+    margin-left: 22px;
+    padding: 10px 5px 10px 30px;
+
+    &:hover {
+      border-radius: variables.$border-radius;
+    }
+  }
+
+  &.active > a::after {
+    content: '';
+    height: 60%;
+    left: 22px;
+    position: absolute;
+    top: 20%;
+    width: 3px;
+  }
+}
+
+km-side-nav-field .sub-menu-item:last-child {
+  margin-bottom: 10px;
+}
+
+.menu-title {
+  border-bottom: 2px solid gray;
+}
+
+.collapsed-sub-menu-item {
+  height: 30px;
+  margin: 0 10px 0 15px;
+  padding: 10px 0 10px 25px;
+  position: relative;
+
+  &:hover {
+    border-radius: variables.$border-radius;
+  }
+
+  &.active::after {
+    content: '';
+    height: 60%;
+    left: 0;
+    position: absolute;
+    top: 20%;
+    width: 3px;
+  }
+}
+
+// go back button side nav bar
+
+.go-back:not(.go-back-collapsed) {
+  bottom: (1.5 * variables.$sidenav-item-margin);
+  height: variables.$sidenav-item-height;
+  position: absolute;
```

Comment:
https://user-images.githubusercontent.com/17727069/204741225-306acfed-7ad3-4a8f-a268-ddd24d0e0049.mp4

I think due to making position: absolute it breaks when you scroll as it overlaps